charts/minecraft-bedrock/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: minecraft-bedrock
-version: 2.4.2
+version: 2.5.0
```

Comment:
Sorry, other chart changes went in so you'll need to manually resolve what git sees as a conflict here.

Reply:
Ok, try that.